WcaOnRails/app/assets/javascripts/associated-events-picker.js

```diff
@@ -3,20 +3,36 @@ $(function() {
     return function() {
       var $eventsFormGroup = $(this).closest('.form-group');
       $eventsFormGroup.find('.event-checkbox input[type="checkbox"]').prop('checked', value);
-      updateEventsSelectedCount($eventsFormGroup);
+      updateEventsInformation($eventsFormGroup);
```

Comment:
Won't this happen for all event pickers on our website, including stuff like the preferred events picker https://www.worldcubeassociation.org/profile/edit?section=preferences?

As I mention in the next comment, I think it would be better for this logic to go in `_register_form.html.erb` below.